app/main.js

```diff
@@ -66,29 +66,29 @@ if (process.env.NODE_ENV === 'development') {
   });
 }
 
-// Listen for a 'resin-update-lock' to either enable, disable or check
+// Listen for a 'balena-update-lock' to either enable, disable or check
 // the update lock from the renderer process (i.e. the app)
-if (electronConfig.ELECTRON_RESIN_UPDATE_LOCK) {
+if (electronConfig.ELECTRON_BALENA_UPDATE_LOCK) {
   const lockFile = require('lockfile');
-  electron.ipcMain.on('resin-update-lock', (event, command) => {
+  electron.ipcMain.on('balena-update-lock', (event, command) => {
     switch (command) {
       case 'lock':
-        lockFile.lock('/tmp/resin/resin-updates.lock', (error) => {
-          event.sender.send('resin-update-lock', error);
+        lockFile.lock('/tmp/balena/balena-updates.lock', (error) => {
```

Comment:
According to https://www.balena.io/docs/learn/deploy/release-strategy/update-locking/ the lockfile path should be `/tmp/balena/updates.lock` ? :eyes: